src/meta/protos/proto/stage.proto

```diff
@@ -74,6 +74,8 @@ message StageInfo {
   optional UserIdentity creator = 8;
 
   FileFormatParams file_format_params = 9;
+
+  bool from_uri = 10;
```

Comment:
Maybe we don't need to add this in proto since it will always be `false`?

Reply:
I think you are right.